File: python/Petshop.py
class Petshop:
    __id = 0
    __name = ""
    __kategori = ""
    __harga = 0

    def __init__(self, id=0, name="", kategori="", harga=0):
        self.__id = id
        self.__name = name
        self.__kategori = kategori
        self.__harga = harga
    # bagian constructor untuk inisialisasi default maupun parameterized di python

    def get_id(self): return self.__id
    # untuk membaca id

    def set_id(self, id): self.__id = id
    # untuk mengubah id

    def get_name(self): return self.__name
    # untuk membaca nama

    def set_name(self, name): self.__name = name
    # untuk mengubah nama

    def get_kategori(self): return self.__kategori
    # untuk membaca kategori

    def set_kategori(self, kategori): self.__kategori = kategori
    # untuk mengubah kategori

    def get_harga(self): return self.__harga
    # untuk membaca harga

    def set_harga(self, harga): self.__harga = harga
    # untuk mengubah harga

    def __str__(self):
        return (f"ID: {self.__id}\n"
                f"Nama: {self.__name}\n"
                f"Kategori: {self.__kategori}\n"
                f"Harga: {self.__harga}\n")
    # untuk menampilkan informasi list petshop dengan tampilan yang mudah untuk dibaca
    
    def __del__(self):
        pass

    # fungsi destructor

# mengubah modifier di python: public (tidak memakai _)
# protected memakai _
# private memakai __

File: php/classes/Petshop.php
<?php
class Petshop {
    protected $id;
    protected $name;
    protected $kategori;
    protected $harga;
    protected $stok;
    protected $foto;

    public function __construct($id = 0, $name = "", $kategori = "", $harga = 0, $stok = 0, $foto = "") {
        $this->id = $id;
        $this->name = $name;
        $this->kategori = $kategori;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->foto = $foto;
    }

    // Getter Methods
    public function getId() {
        return $this->id;
    }

    public function getName() {
        return $this->name;
    }

    public function getKategori() {
        return $this->kategori;
    }

    public function getHarga() {
        return $this->harga;
    }

    public function getStok() {
        return $this->stok;
    }

    public function getFoto() {
        return $this->foto;
    }

    // Setter Methods
    public function setId($id) {
        $this->id = $id;
        return $this;
    }

    public function setName($name) {
        $this->name = $name;
        return $this;
    }

    public function setKategori($kategori) {
        $this->kategori = $kategori;
        return $this;
    }

    public function setHarga($harga) {
        $this->harga = $harga;
        return $this;
    }

    public function setStok($stok) {
        $this->stok = $stok;
        return $this;
    }

    public function setFoto($foto) {
        $this->foto = $foto;
        return $this;
    }
}
?>